WooCommerce/WooCommerceTests/Universal Links/UniversalLinkRouterTests.swift

```diff
@@ -109,7 +137,7 @@ final class UniversalLinkRouterTests: XCTestCase {
                 queryItem
             ]
 
-        let route = MockRoute(subPath: subPath, performAction: { parameters in
+        let route = MockRoute(handledSubpaths: [subPath], performAction: { _, parameters in
```

Comment:
We don't seem to need this parameter for the test:
```suggestion
        let route = MockRoute(handledSubpaths: [subPath], performAction: { _, _ in
```